public/app/features/dashboard/components/AddPanelWidget/AddPanelWidget.tsx

```diff
@@ -194,33 +194,33 @@ interface AddPanelWidgetHandleProps {
   onCancel: (e: React.MouseEvent<HTMLButtonElement>) => void;
   onBack?: () => void;
   children?: string;
+  styles: AddPanelStyles;
 }
 
-const AddPanelWidgetHandle: React.FC<AddPanelWidgetHandleProps> = ({ children, onBack, onCancel }) => {
-  const theme = useTheme();
-  const styles = getAddPanelWigetHandleStyles(theme);
+const AddPanelWidgetHandle: React.FC<AddPanelWidgetHandleProps> = ({ children, onBack, onCancel, styles }) => {
   return (
-    <div className={cx(styles.handle, 'grid-drag-handle')}>
-      <div className={styles.backButton}>
-        {onBack && (
-          <IconButton name="arrow-left" onClick={onBack} surface="header" size="xl" className={styles.noMargin} />
-        )}
-      </div>
-
+    <div className={cx(styles.headerRow, 'grid-drag-handle')}>
       {children && <span>{children}</span>}
-      <IconButton name="times" onClick={onCancel} surface="header" className={styles.pushRight} />
+      {onBack && (
+        <div className={styles.backButton}>
+          <IconButton name="arrow-left" onClick={onBack} surface="header" size="md" />
+        </div>
+      )}
+      <div className="flex-grow-1" />
+      <IconButton name="times" onClick={onCancel} surface="header" />
     </div>
   );
 };
 
-const getStyles = stylesFactory((theme: GrafanaTheme) => {
+const getStyles = (theme: GrafanaTheme) => {
```

Comment:
@kaydelaney tip, stylesFactory is not needed when you use `useStyles(getStyles)`,   useStyles will memoize the call the same way stylesFactory does